### ISSUE TITLE
8.2.1 breaks parsing bitbake functions resulting in oelint.spaces.linebeginning false-positives

### ISSUE THREAD

**Issue by skycaptain** [priv-kweihmann/oelint-parser#308]
The oelint-parser v8.2.1 has an issue with parsing shell function blocks, resulting in false warnings for "oelint.spaces.linebeginning":

```bitbake
SUMMARY = "A simple example recipe for Yocto"
DESCRIPTION = "A description longer than the summary"
HOMEPAGE = "https://example.com"
LICENSE = ""
LIC_FILES_CHKSUM = ""

SRC_URI = ""

do_install() {
    install -d ${D}${datadir}/my-example
    cp -r ${S}/* ${D}${datadir}/my-example/
}
```

```
Parsing [redacted]/test.bb
test.bb:11:warning:oelint.spaces.linebeginning:Line shall not begin with a space [branch:true]
```

I get no such warnings with oelint-parser v8.2.0:

```
Parsing [redacted]/test.bb
```

What's interesting is that the `TestClassOelintSpacesLineBeginning` tests of oelint-adv seem to pass with v8.2.1.
